draft-irtf-panrg-path-properties.md

```diff
@@ -136,6 +138,10 @@ Assessed property:
 : An approximate calculation or assessment of the value of a property. An assessed property includes the reliability of the calculation or assessment. The notion of reliability depends on the property.
 For example, a path property based on an approximate calculation may describe the expected median one-way latency of packets sent on a path within the next second, including the confidence level and interval. A non-numerical assessment may instead include the likelihood that the property holds.
 
+Target property:
+: An objective that is set for a property over a path element, subpath, or path.
+Note that a target property can be set for observable properties, such as one-way delay, but also for properties that are not (easily) observable, such as inclusion of certain nodes on a path.
```

Comment:
I wonder if this section should say "observed" rather than "observable" properties, as that's the defined term, and I think it's obvious enough that OWD being an "observed" property just means that it can be observed, given appropriate measurement tools etc.
Then, the second part could become "properties that cannot be observed by the entity setting the target" - surely, *somebody* could observe whether the node is on the path, but the entity setting the target may not have that insight because the node is not in their domain. (No need to get that specific in the definition, I just wanted to illustrate how I think about it - "inclusion of certain nodes on a path" is not inherently non-observable, it just might be non-observable to a certain entity, in analogy to the SLE in [Section 4.1 of draft-ietf-teas-ietf-network-slices](https://www.ietf.org/archive/id/draft-ietf-teas-ietf-network-slices-14.html#name-objectives-for-ietf-network))

Reply:
I like both suggestions.
Also, discussing that some entity may be able to observe a property while another entity cannot is more in line with our definition of a path (i.e., partial visibility and different levels of abstraction).

What do you think about the following change?

```suggestion
Note that a target property can be set for observed properties, such as one-way delay, but also for properties that cannot be observed by the entity setting the target, such as inclusion of certain nodes on a path.
```

Reply:
This looks good to me, thank you!